satellite.py

```diff
@@ -96,3 +149,43 @@
                   if chosen]
 
 print(constellations)
+
+# don't consider constellations with average score less than score_threshold
+score_threshold = .4
+
+bqm = dimod.BinaryQuadraticModel.empty(dimod.BINARY)
```

Comment:
Why not use the more efficient `dimod.AdjVectorBQM` or similar here?

Reply:
Yes indeed. Thanks.